### PR TITLE
Class methods should be non enumerable

### LINKED ISSUE #1873: Class methods should be non-enumerable
Not sure about getters/setters...

Causing us problems at https://github.com/Igalia/streams/commit/83222c5b08f26ff37fbea519485a7bcc34f4a740

### PR DESCRIPTION
**Pull request by arv**
Fixes #1873

### CODE REVIEW

**Comment by arv** [comment]
@domenic PTAL

File: test/feature/NumericLiteral/Simple.js [hunk 1]
```diff
@@ -83,6 +83,19 @@
     static get 0O12() {}
     static set 0O13(v) {}
   }
-  assertArrayEquals(['0', '1', '2', '6', '7', '8'], Object.keys(C.prototype));
-  assertArrayEquals(['3', '4', '5', '9', '10', '11'], Object.keys(C));
-})();
\ No newline at end of file
+
+  assert.isTrue(C.prototype.hasOwnProperty('0'));
+  assert.isTrue(C.prototype.hasOwnProperty('1'));
```
Comment (domenic) [inline test/feature/NumericLiteral/Simple.js]: This change reduces coverage by allowing extra properties to sneak in. Maybe use gOPNs instead of O.keys?

**Comment by domenic** [comment]
LGTM with a test style nit. Interesting that we made getters nonenumerable.... Not sure I agree with that, but oh well.

Thanks so much for the quick response!

File: test/feature/NumericLiteral/Simple.js (hunk 1 above)
Comment (arv) [inline test/feature/NumericLiteral/Simple.js]: I had `getOwnPropertyNames` first but that includes `arguments` and `caller` which is wrong according to ES6.

**Comment by johnjbarton** [comment]
I double checked this PR should be in npm. Please let me know if that is not what you see.

**Pull request merged**